fv3core/stencils/dyn_core.py

```diff
@@ -174,7 +174,21 @@ def compute(state, comm):
         )
     n_con = get_n_con()
 
+    # "acoustic" loop
+    # called this because its timestep is usually limited by horizontal sound-wave
+    # processes. Note this is often not the limiting factor near the poles, where
+    # the speed of the polar night jets can exceed two-thirds of the speed of sound.
```

Comment:
It's not particularly important for content reasons, I just know this was one of the things that really stuck with me from the documentation about the acoustic loop, which I think will help with recall about acoustic timestep restrictions for the acoustic loop. I mostly included it here so that it can make its way into docstrings or sphinx docs down the line.